packages/ui/src/components/Tag/Tag.tsx

```diff
@@ -3,32 +3,32 @@ import { cva } from "class-variance-authority";
 import { twMerge } from "@/utils";
 
 export const tagStyles = cva(
-  ["flex items-center justify-center", "rounded-6 font-semibold border"],
+  [
+    "flex items-center justify-center w-fit",
+    "font-medium shadow-secondary-button",
+  ],
   {
     variants: {
       size: {
-        sm: ["h-8 px-2 text-sm leading-4"],
-        xs: ["h-6 px-[6px] text-2xs leading-3"],
+        md: ["h-8 px-2 text-base leading-5 rounded-[10px]"],
+        sm: ["h-6 px-1.5 text-sm leading-4 rounded-8"],
+        xs: ["h-5 px-1.5 text-xs leading-3 rounded-6"],
       },
       colorScheme: {
-        primary: ["bg-surface-primary-main text-surface-white border-black-12"],
-        secondary: [
-          "bg-surface-primary-accent-2 text-text-primary-em border-transparent",
+        primary: ["bg-surface-primary-med-em text-text-inverse-black-neutral"],
+        "primary-light": [
+          "bg-surface-primary-base-em-alpha text-text-primary-high-em",
         ],
-        tertiary: [
-          "bg-surface-surface-0 text-text-high-em border-outline-low-em",
+        "primary-outline": [
+          "bg-surface-surface-white-smoke-0 text-text-primary-high-em border border-outline-primary-alpha",
```

Comment:
Same here, maybe we can use main.

Reply:
is the same, the variant is `primary`. 
Here we have 2 descents of primary: outline and light, I added primary to create the relationship but maybe we can drop it?

Reply:
same as above